src/crystal/event_loop.cr

```diff
@@ -1,28 +1,56 @@
 require "./event"
 
-module Crystal::EventLoop
-  @@eb = Crystal::Event::Base.new
-  @@dns_base : Crystal::Event::DnsBase?
+class Thread
+  @eb : Crystal::Event::Base?
+  @dns_base : Crystal::Event::DnsBase?
+  @loop_fiber : Fiber?
 
-  def self.after_fork
-    @@eb.reinit
+  # :nodoc:
+  def eb : Crystal::Event::Base
+    @eb ||= Crystal::Event::Base.new
```

Comment:
Rename `@eb` to `@event_base`.
A getter can be used (and for `dns_base`)
`getter event_base : Crystal::Event::Base { Crystal::Event::Base.new }`